site/styles.css

```diff
@@ -391,6 +391,9 @@ body, html, #app {
       font-weight: bold;
       color: var(--text-red); }
 
+    .message-inline-code {
+      white-space: pre; }
+
```

Comment:
So what I'm getting is that we support `\n` in `` `abc` ``-style code blocks, and that ```` ``` ```` code blocks just aren't a thing. I wonder if this should be changed - using two line breaks in a `` `code` `` element `isn't

supported` in Markdown.

In any case, this class name definitely dosen't make sense!

```
blah:

`
asdf
baz
batr
`
```

^ This definitely isn't inline code, is it :)

Reply:
>So what I'm getting is that we support \n in `abc`-style code blocks,

Unintentional, but yes. And there's not really much point in fixing it in my opinion.

> ``` code blocks just aren't a thing

They are-- try them.

>  this class name definitely dosen't make sense!

Will fix

Reply:
>> triple-backslash code blocks just aren't a thing
>
> They are-- try them.

Hm? I don't see any code written to handle ```` ``` ````. Sure, using that string of characters *works*, but I don't think *that's* intentional - rather it just ignores the first two ticks and treats the third as the beginning of the "inline" code-block.

Reply:
100% intentional 😜 

I _thought_ I implemented it at some point but clearly not.